tests/2018-03-01-preview/Microsoft.Storage.tests.json

```diff
@@ -0,0 +1,186 @@
+{
+  "tests": [
```

Comment:
>tests [](start = 3, length = 5)

some test cases are failing. please fix them.

Reply:
@TianoMS 

I see all checks pas below. And I have run the test on my local machine and all pass.
Would you please show how to repro the test failure?
And what's the test failure message?

Thanks!

Reply:
my bad. the test cases are passing.

Reply:
@TianoMS 
Could you give more details for the test failure? like how to repro, and the test failure message?
It seems I can't repro.

Reply:
There's no test failure.. I was wrong. Something went wrong when I merge with your branch. So you are all good.